### PR TITLE
Connect Wallet Visual Update

### PR DESCRIPTION
**Pull request by 0xFirekeeper**
Little visual update, colors outlines and added balance + current network.
I like how this looks now, but we can wait until we get updated chain icons to display if you like. 
As for switching networks, I'm not sure it's the best idea for a game connect wallet prefab as most games will be on a specific chain + will require tinkering with the prefab to remove it.
Switching network as a separate prefab later on perhaps?

### CODE REVIEW

**Comment by 0xFirekeeper** [comment]
Will add Prefab_SwitchNetwork as a separate prefab and append it to connect wallet so people can just set it inactive if they don't want it. It'll be on a simple `>` icon near current network.

**Comment by 0xFirekeeper** [comment]
- Chain enum instead of strings in the editor, has chain ID built in, and a dictionary in case you need the string identifier, in ThirdwebManager
- Added switching networks to the connect wallet prefab, there's a bool on whether you want to support it or not which auto turns off its UI
- Added supported networks to ThirdwebManager, when switching network only those will display
- Prefab_NFTLoader now scales with the screen
- Other minor improvements

File: Assets/Thirdweb/Examples/Scripts/ThirdwebManager.cs [hunk 1]
```diff
@@ -1,10 +1,49 @@
...
+    Fantom = 250,
+    FantomTestnet = 4002,
+    Avalanche = 43114,
+    AvalancheTestnet = 43113,
+    Optimism = 10,
+    OptimismGoerli = 420,
+    Arbitrum = 42161,
+    ArbitrumGoerli = 421613,
+    Binance = 56,
+    BinanceTestnet = 97
```
Comment (joaquim-verges) [inline Assets/Thirdweb/Examples/Scripts/ThirdwebManager.cs:21]: niceee

**Review by joaquim-verges — COMMENTED** [review]
couple of visual things:

- on the connected state, the button + fonts feel a bit too small
- the background color of the connected state is too close to the header color in the scene, making it hard to see (maybe needs a brighter border?)
- on the nft loader, if you resize too small, then completely disappears, should probably have a min width of 1 column worth?

**Comment by 0xFirekeeper** [comment]
Updated and restructured a little

**Comment by joaquim-verges** [comment]
@0xFirekeeper a bit too much to have all the chain icons i think, can we just keep the ones from the chains we support, and maybe only the high res version for each?

**Comment by joaquim-verges** [comment]
last thing i noticed too : on the dropdown when con
Screen Shot 2023-02-02 at 7 54 52 PM
nected, the current network still has the placeholder icon

**Comment by 0xFirekeeper** [comment]
Ah good catch, fixing

**Comment by 0xFirekeeper** [comment]
Should be good now

**Pull request merged**